README.md

```diff
@@ -166,25 +166,25 @@ load tests. For example, pass the `-h` flag to the `simple` example,
 
 ```
 client 0.7.5
-CLI options available when launching a Goose loadtest
+CLI options available when launching a Goose load test
 
 USAGE:
     simple [FLAGS] [OPTIONS]
 
 FLAGS:
-    -h, --help                Prints help information
-    -l, --list                Shows list of all possible Goose tasks and exits
-    -g, --log-level           Log level (-g, -gg, -ggg, etc.)
-        --manager             Enables manager mode
-        --no-hash-check       Ignore worker load test checksum
-        --no-stats            Don't print stats in the console
-        --no-sticky-follow    Prevent load test client from following redirect of base_url with subsequent requests
-        --only-summary        Only prints summary stats
-        --reset-stats         Resets statistics once hatching has been completed
-        --status-codes        Includes status code counts in console stats
-    -V, --version             Prints version information
-    -v, --verbose             Debug level (-v, -vv, -vvv, etc.)
-        --worker              Enables worker mode
+    -h, --help             Prints help information
+    -l, --list             Shows list of all possible Goose tasks and exits
+    -g, --log-level        Log level (-g, -gg, -ggg, etc.)
+        --manager          Enables manager mode
+        --no-hash-check    Ignore worker load test checksum
+        --no-stats         Don't print stats in the console
+        --only-summary     Only prints summary stats
+        --reset-stats      Resets statistics once hatching has been completed
+        --status-codes     Includes status code counts in console stats
+        --sticky-follow    Client follows redirect of base_url with subsequent requests
```

Comment:
@LionsAd @nnewton it turns out this feature was a misunderstanding on my part, and Locust does NOT do this by default. 

In any case 1) there's good cleanup in this PR, and 2) I believe there's still a use-case -- as such, I'm making this functionality available behind a flag:  only if a load test is started with `--sticky-follow` will the base_url follow a redirect for subsequent requests. As it's not the default it shouldn't trip anyone up during normal testing, and I believe this addresses everyone's concerns.